Models/Memory.cs

```diff
@@ -143,9 +143,9 @@ public void SetBytesUsed(MemoryByteUsage usage, int size, int offset = 0, object
                     MemorySection sec = Sections.Single(s => s.Address == secAddr);
                     int byteIndex = paraAddr - secAddr;
                     sec.Bytes[byteIndex].SetByteUsed(usage, usedBy);
-                }catch(Exception e){
-                    if (e is InvalidOperationException){
-                        return;
+                }catch(Exception ex){
```

Comment:
Please use something like this:
```C#
try {

} catch(InvalidOperationException ex) {
 throw new OutOfMemoryException(ex);
}
```

This will only catch Exceptions of type InvalidOperation.
(Make sure you also use the InnerException)